src/sentry/auth/authenticators/u2f.py

```diff
@@ -116,8 +136,34 @@ def activate(self, request):
 
         return ActivationChallengeResult(challenge=challenge)
 
-    def validate_response(self, request, challenge, response):
+    def validate_response(self, request, challenge, response, is_webauthn_signin_ff_enabled):
         try:
+            if is_webauthn_signin_ff_enabled:
```

Comment:
you can have it so that this block isn't nested in an if by using a return early pattern

Reply:
don't we have to check the FF in this case? we don't want the nested logic to be ran when the FF isn't on

Reply:
can you just say:
```
if is_webauthn_signin_ff_enabled:
    u2f.complete_authentication(challenge, response, self.u2f_facets)
```
?

Reply:
ah ur right, my brain did not work for a second